src/librustc/ty/sty.rs

```diff
@@ -1894,6 +1894,14 @@ impl<'tcx> TyS<'tcx> {
         }
     }
 
+    /// Panics if called on any type other than `PhantomData<T>`.
+    pub fn phantom_ty(&self) -> Ty<'tcx> {
```

Comment:
Is this used somewhere? I didn't find a use

Reply:
Yep, you're right; I used it for a hack earlier, but that's no longer necessary, so I can remove it.